core/common/io/src/test/java/org/eclipse/rdf4j/common/lang/FileFormatTest.java

```diff
@@ -0,0 +1,27 @@
+/*******************************************************************************
+ * Copyright (c) $2022 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *******************************************************************************/
+package org.eclipse.rdf4j.common.lang;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.ExpectedException;
+
+public class FileFormatTest {
+
+    @Rule
+    public ExpectedException thrown = ExpectedException.none();
+
+    @Test
+    public void testEmptyMimeType() {
+        thrown.expect(IllegalArgumentException.class);
+        new FileFormat("PLAIN TEXT", new ArrayList<String>(), null, Arrays.asList("txt"));
+    }
```

Comment:
We've switched over to JUnit 5 a while back and are slowly migrating our existing tests over to it. Do you want to rewrite this as a JUnit 5 test?

Reply:
Sure I can do that. Does [this](https://pastebin.com/DxmTfit8) look good to you?